api/v1/db/aws/aws-operations.js

```diff
@@ -37,13 +36,12 @@ const bucketExists = (bucket = thisBucket) => new Promise((resolve, reject) => {
 /**
  * @summary Checks if an object exists in a bucket
  * @function
- * @param key The key of the object to be checked
- * @param bucket The bucket where the key will be searched
+ * @param {string} key The key of the object to be checked
+ * @param {string} bucket The bucket where the key will be searched
  * @returns {Promise} Promise object represents a boolean indicating if the key exists or not
  */
 const objectExists = (key, bucket = thisBucket) => new Promise((resolve, reject) => {
```

Comment:
This method can be removed since `headObject` won't be used.

Reply:
I think this method will still be needed here https://github.com/osu-mist/notes-api/blob/80204216476ce3e2e0e49162496574f98a95f703/api/v1/db/aws/notes-dao.js#L145-L149 it checks for the existence of the student directory before trying to create it. Concurrency shouldn't matter here since the object is going to be empty and the metadata should be the same for any putObject request for it.